test/evergreen.yml

```diff
@@ -261,36 +267,24 @@ functions:
           cd wiredtiger.github.com
 
           # Go through each supported WiredTiger Github branch to stage the doc changes.
-          #   Please note the "branch" variable used here is a normal shell variable to store the name of
-          #   a WiredTiger Github branch, while the "branch_name" used above is an Evergreen built-in variable
-          #   to store the name of the branch being tested by the Evergreen project.
-          for branch in mongodb-4.0 mongodb-4.2 mongodb-4.4 mongodb-5.0 develop ; do
-            echo "[Debug] Copying over doc directory for branch $branch ..."
-            rsync -avq ../docs-$branch/ $branch/
-
-            # Commit and push the changes (if any).
-            git add $branch > /dev/null
-          done
+          for branch in develop mongodb-5.0 mongodb-4.4 mongodb-4.2 mongodb-4.0 ; do
 
-          git commit -m "Update auto-generated develop docs" \
-                     --author="svc-bot-doc-build <svc-wiredtiger-doc-build@10gen.com>" > /dev/null
-    # Use separate shell.exec with "silent" directive to avoid exposing credentail in task log.
-    - command: shell.exec
```

Comment:
I can see the merge of the 2 shell.exec commands is attempting to save the duplicated logic checking. The reason it's been split into 2 initially was we hoped to avoid exposing the api token, and only silence the "required" part (i.e. the "git push" command), because making the shell script verbose is generally a useful practice, e.g. when the shell script fails at the "git commit" step we want the log dump to confirm that.

Reply:
I think it's more useful (and better practice) to echo a message to the log file when a command fails. This allows us to include specific information that might be helpful in diagnosing the issue.

Reply:
The push command is now done silently here 15dea4d0e62cb751be2a84a8d62199ccfcfe4360